src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/course/domain/CourseService.kt

```diff
@@ -82,7 +82,7 @@ class CourseService(
 
   private fun audienceStrings(audience: String): List<String> = audience.split(',').map(String::trim)
 
-  fun replaceAllPrerequisites(replacements: List<NewPrerequisite>): List<LineMessage> {
+  fun uploadPrerequisitedCsv(replacements: List<NewPrerequisite>): List<LineMessage> {
```

Comment:
Again, I wonder if the `replace` here is useful for readers to understand that it's blowing them away.